src/data/list/join.lean

```diff
@@ -0,0 +1,128 @@
+/-
+Copyright (c) 2014 Parikshit Khanna. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Parikshit Khanna, Jeremy Avigad, Leonardo de Moura, Floris van Doorn, Mario Carneiro
+-/
```

Comment:
This file seems to be:
* largely ee8451b9f60deb741e0480789f4d0406c553857d
* one lemma from 604699b14123e1e9ad93eb48340c31c327d9e70a
* two lemmas from c13a7711450835bd8e53c128f12909615a5a1d90
* one lemma from d95bff068a5373593a9a3d779f75bafe3c2868e7

I don't see `Parikshit Khanna` as an author of any of those, so my guess would probably be
```suggestion
/-
Copyright (c) 2017 Mario Carneiro. All rights reserved.
Released under Apache 2.0 license as described in the file LICENSE.
Authors: Sébastien Gouëzel, Floris van Doorn, Mario Carneiro
-/
```